### DISCUSSION TITLE
Setting up webhooks for Google Cloud Source Repository

### DISCUSSION THREAD

**Discussion by dsvgmbh** [codemagic-ci-cd/codemagic-docs#2313] [Ideas]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

We are using Google Cloud Source Repository. Any guideline to use codemagic with google source repo via webhook. on docs i can not find any hints.

https://docs.codemagic.io/yaml-running-builds/webhooks/

### Expected results

use webhooks with Google Cloud Source Repository

### Actual results

no documentation or any guideline

**Comment by mikhail-tokarev** [comment]
Hi

Sorry for the late reply! unfortunately we don't support webhooks from Google Cloud Source at this moment. I will move your question to feature ideas topic to see if more users will request it.

Thank you and my apologies again for the late response.
